ui/src/app/shared/workflow/node/run/node.run.param.html

```diff
@@ -17,7 +17,7 @@
     </div>
     <div class="content">
         <div class="ui form">
-            <ng-container *ngIf="_nodeToRun">
+            <ng-container *ngIf="_nodeToRun && _nodeToRun.context.default_pipeline_parameters.length > 0 ">
```

Comment:
`_nodeToRun && _nodeToRun.context && _nodeToRun.context.default_pipeline_parameters && _nodeToRun.context.default_pipeline_parameters.length > 0`

Reply:
we don't do that the line below: 
`                <app-parameter-list [project]="project" [parameters]="_nodeToRun.context.default_pipeline_parameters"
`